python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -216,6 +216,12 @@ def print_detailed(self):
         """
         return self._print_detailed()
 
+    def dump_as_json(self):
```

Comment:
This returns rather than dumping to a file right? Maybe `return_as_json`? or `dumps_as_json` if you prefer to match the json module naming?

Reply:
Given that `dump_as_json()` is part of the 0.17 release, renaming it would be a breaking change.

Reply:
I renamed `dump_as_json()` to `get_json()`, to be consistent with `get_summary_text()` and `get_detailed_text()`.